configure.ac

```diff
@@ -514,6 +520,9 @@ if test "$cross_compiling" = yes; then
 	*-*-vxworks*)
 		_host_cpu=$host_cpu
 		;;
+	wasm32-*-* | wasm64-*-*)
```

Comment:
Same question about whether `wasm32-wasi` is supported while having the two `-`.

Reply:
Same answer, it's already canonicalized.